docs/getting-started-guides/ubuntu/installation.md

```diff
@@ -9,7 +9,22 @@ title: Setting up Kubernetes with Juju
 Ubuntu 16.04 introduced the [Canonical Distribution of Kubernetes](https://www.ubuntu.com/cloud/kubernetes), a pure upstream distribution of Kubernetes designed for production usage. This page shows you how to deploy a cluster.
 {% endcapture %}
 
-Out of the box it comes with the following components on 9 machines:
+{% capture prerequisites %}
+- A working [Juju client](https://jujucharms.com/docs/2.2/reference-install); this does not have to be a Linux machine, it can also be Windows or OSX.
+- A [supported cloud](#cloud-compatibility).
+  - Bare Metal deployments are supported via [MAAS](http://maas.io). Refer to the [MAAS documentation](http://maas.io/docs/) for configuration instructions.
+  - OpenStack deployments are currently only tested on Icehouse and newer.
+- Network access to the following domains
+  - *.jujucharms.com
```

Comment:
You need to escape this asterisk (*) character -- otherwise the output displays italic until the next asterisk at line 51 (which you also need to escape)